tests/functional/filter/cram/filter-exclude-include.t

```diff
@@ -10,6 +10,6 @@ Force include one South American record by country to get two total records.
   >  --metadata "$TESTDIR/../data/metadata.tsv" \
   >  --exclude-where "region=South America" "region=North America" "region=Southeast Asia" \
   >  --include-where "country=Ecuador" \
-  >  --output-strains filtered_strains.txt > /dev/null
+  >  --output-strains filtered_strains.txt &>/dev/null
```

Comment:
With the current design (especially the chosen names, where we use `print_err` for non-error logging) it's going to be easy for regressions to slip back in. I suggest changing the tests to only redirect stderr to /dev/null.

Reply:
Good suggestion - done in 201e1de7c02c209c98e2309a274c125d2067ac2a.

I've also added 9ab80b55709a5b32ee157dc72edd50d20c6d29f7 to clarify `print_err` a bit. Maybe we can rename in another PR - can you think of a better name?

Reply:
Thanks! 

> can you think of a better name?

I'm sure others will have opinions here. `log` comes to mind.